reposilite-frontend/src/components/browser/Card.vue

```diff
@@ -150,13 +173,19 @@ export default {
       return createToast('Copied snippet', { type: 'info' })
     }
 
+    const toggleConfigDropdown = () => {
+      document.querySelector(".config-dropdown ul").style.display = dropdownOpen ? 'none' : 'block';
```

Comment:
I think it might be replaced with `v-if="dropdownOpen"` and exported `ref` on `dropdownOpen`. Are you familiar with new reactive approach in Vue 3? It's not a priority for me, so if you feel you're not able to do it, just let me know.

Reply:
Done, Updated as you mentioned.